build.sh

```diff
@@ -87,98 +93,71 @@ NEST_RESULT=$(readlink -f $NEST_RESULT)
 mkdir "$NEST_VPATH" "$NEST_RESULT"
 mkdir "$NEST_VPATH/reports"
 
-if [ "$xSTATIC_ANALYSIS" = "1" ] ; then
-  echo
-  echo "+ + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + +"
-  echo "+               S T A T I C   C O D E   A N A L Y S I S                       +"
-  echo "+ + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + + +"
-
-  echo "MSGBLD0010: Initializing VERA++ static code analysis."
-  # Add the NEST profile to the VERA++ profiles.
-  sudo cp ./extras/vera++.profile /usr/lib/vera++/profiles/nest
-  echo "MSGBLD0020: VERA++ initialization completed."  
-
-  if [ ! -f "$HOME/.cache/bin/cppcheck" ]; then
-    echo "MSGBLD0030: Installing CPPCHECK version 1.69."
-    # Build cppcheck version 1.69.
-    git clone https://github.com/danmar/cppcheck.git
-    cd cppcheck
-    git checkout tags/1.69
-    mkdir -p install
-    make PREFIX=$HOME/.cache CFGDIR=$HOME/.cache/cfg HAVE_RULES=yes install
-    cd ..
-    echo "MSGBLD0040: CPPCHECK installation completed."
-
-    echo "MSGBLD0050: Installing CLANG-FORMAT."
-    wget --no-verbose http://llvm.org/releases/3.6.2/clang+llvm-3.6.2-x86_64-linux-gnu-ubuntu-14.04.tar.xz
-    tar xf clang+llvm-3.6.2-x86_64-linux-gnu-ubuntu-14.04.tar.xz
-    # Copy and not move because '.cache' may aleady contain other subdirectories and files.
-    cp -R clang+llvm-3.6.2-x86_64-linux-gnu-ubuntu-14.04/* $HOME/.cache
-    echo "MSGBLD0060: CLANG-FORMAT installation completed."
```

Comment:
I agree with @Silmathoron. Our [C++ Coding Guidelines](https://nest.github.io/nest-simulator/coding_guidelines_c++#tooling) explicitly point out that we require clang-format 3.6.

Can you please double-check if all checks run properly using this old version? One way to check would probably be to manually check all our source code files in a TravisCI debug build.